### DISCUSSION TITLE
protocol error

### DISCUSSION THREAD

**Discussion by Ryzz3nn** [ORelio/Minecraft-Console-Client#2004] [Q&A]
![image](https://user-images.githubusercontent.com/47952472/165099985-f80b8362-26c6-4660-b9ba-31342edf2056.png)

so ive noticed that if the counter that shows how many players are connected it says that the protocol is not supported and wont connect, is there a work around for this

**Comment by ORelio** [comment]
Set the version you want to use manually in config:
https://github.com/MCCTeam/Minecraft-Console-Client/blob/17dbe10ff3c3fc344e7f6bfa82607340eebeb005/MinecraftClient/Resources/config/MinecraftClient.ini#L26
